sections/dashboard/Overview/Overview.tsx

```diff
@@ -23,48 +33,89 @@ enum MarketsTab {
 	const { t } = useTranslation();
 
+	const { useExchangeRatesQuery, useSynthsBalancesQuery } = useSynthetixQueries();
+
 	const futuresMarketsQuery = useGetFuturesMarkets();
 	const futuresMarkets = futuresMarketsQuery?.data ?? [];
 
+	const { network } = Connector.useContainer();
+	const markets = futuresMarkets.map(({ asset }) => getMarketKey(asset, network.id));
+	const portfolioValueQuery = useGetCurrentPortfolioValue(markets);
+	const portfolioValue = portfolioValueQuery?.data ?? null;
+
 	const futuresPositionQuery = useGetFuturesPositionForAccount();
 	const futuresPositionHistory = futuresPositionQuery?.data ?? [];
 
+	const exchangeRatesQuery = useExchangeRatesQuery();
+	const exchangeRates = exchangeRatesQuery.isSuccess ? exchangeRatesQuery.data ?? null : null;
+
+	const walletAddress = useRecoilValue(walletAddressState);
+	const synthsBalancesQuery = useSynthsBalancesQuery(walletAddress);
+	const synthBalances =
+		synthsBalancesQuery.isSuccess && synthsBalancesQuery.data != null
+			? synthsBalancesQuery.data
+			: null;
+
 	const [activePositionsTab, setActivePositionsTab] = useState<PositionsTab>(PositionsTab.FUTURES);
 	const [activeMarketsTab, setActiveMarketsTab] = useState<MarketsTab>(MarketsTab.FUTURES);
 
+	const totalSpotBalancesValue = formatCurrency(
+		Synths.sUSD,
+		wei(synthBalances?.totalUSDBalance ?? zeroBN),
+		{
+			sign: '$',
+		}
+	);
+
+	const totalFuturesPortfolioValue = formatCurrency(Synths.sUSD, wei(portfolioValue ?? zeroBN), {
+		sign: '$',
+	});
+
+	const totalShortsValue = formatCurrency(Synths.sUSD, wei(zeroBN), {
+		sign: '$',
+	});
+
 	const POSITIONS_TABS = useMemo(
 		() => [
 			{
 				name: PositionsTab.FUTURES,
 				label: t('dashboard.overview.positions-tabs.futures'),
 				badge: futuresPositionQuery?.data?.length,
 				active: activePositionsTab === PositionsTab.FUTURES,
+				detail: totalFuturesPortfolioValue,
 				onClick: () => {
 					setActivePositionsTab(PositionsTab.FUTURES);
 				},
 			},
 			{
-				name: PositionsTab.SHORTS,
-				label: t('dashboard.overview.positions-tabs.shorts'),
-				badge: 3,
-				disabled: true,
-				active: activePositionsTab === PositionsTab.SHORTS,
+				name: PositionsTab.SPOT,
+				label: t('dashboard.overview.positions-tabs.spot'),
+				active: activePositionsTab === PositionsTab.SPOT,
+				detail: totalSpotBalancesValue,
 				onClick: () => {
-					setActivePositionsTab(PositionsTab.SHORTS);
+					setActivePositionsTab(PositionsTab.SPOT);
 				},
 			},
 			{
-				name: PositionsTab.SPOT,
-				label: t('dashboard.overview.positions-tabs.spot'),
+				name: PositionsTab.SHORTS,
+				label: t('dashboard.overview.positions-tabs.shorts'),
 				badge: 3,
 				disabled: true,
-				active: activePositionsTab === PositionsTab.SPOT,
+				active: activePositionsTab === PositionsTab.SHORTS,
+				detail: totalShortsValue,
 				onClick: () => {
-					setActivePositionsTab(PositionsTab.SPOT);
+					setActivePositionsTab(PositionsTab.SHORTS);
 				},
 			},
```

Comment:
@platschi Do you think we should leave this shorts value in here for now or just remove it? At the moment this will show $0 even if a trader has a short open, so it might be misleading.

Reply:
True, makes sense to hide the value for now for Shorts and just leave the button disabled with the "Shorts" text so users know it's coming eventually.